src/actions/reports.js

```diff
@@ -1,16 +1,37 @@
 import { tokenSelector } from 'selectors/authSelector';
-import { postExperiencesReports as postExperiencesReportsApi } from 'apis/reportsExperiencesApi';
+import {
+  createExperienceReportApi,
+  createSalaryWorkTimeReportApi,
+} from 'apis/reports';
 
-export const createReport = ({ experienceId, body }) => (
-  dispatch,
+export const createExperienceReport = ({ id, reason, reasonCategory }) => (
+  _,
   getState,
 ) => {
   const state = getState();
   const token = tokenSelector(state);
+  return createExperienceReportApi({
+    id,
+    reason,
+    reasonCategory,
+    token,
+  }).catch(error => {
+    throw error;
+  });
```

Comment:
這個檔案的所有 catch + throw 都可以省略
```suggestion
  });
```

Reply:
OK，我看到 graphqlClient 有實作 throw error 了，感恩~